src/conversions.cc

```diff
@@ -34,7 +34,7 @@ void InitConversions(Local<Object> exports) {
     v8::Local<v8::Object> bufferView;
     bufferView = node::Buffer::New(Isolate::GetCurrent(), point_transfer_buffer, 0, 2 * sizeof(uint32_t)).ToLocalChecked();
     auto js_point_transfer_buffer = node::Buffer::Data(bufferView);
-  #elif V8_MAJOR_VERSION >= 8
+  #elif (V8_MAJOR_VERSION > 8 || (V8_MAJOR_VERSION == 8 && V8_MINOR_VERION > 3))
```

Comment:
This change is not related to Node 19, but I couldn't resist including it. This version check is more correct and I noticed this issue because (if I recall correctly) I was prebuilding node-tree-sitter for some Electron version which has V8 version 8.0 8.1 or 8.2 and it failed on this line.